rust/agents/relayer/src/msg/metadata/multisig/base.rs

```diff
@@ -87,18 +78,12 @@ pub trait MultisigIsmMetadataBuilder: AsRef<BaseMetadataBuilder> + Send + Sync {
                         .collect();
                     Ok(ethers::abi::encode(&proof_tokens))
                 }
-                MetadataToken::Validators => {
-                    let validator_tokens: Vec<Token> = validators
-                        .iter()
-                        .map(|x| Token::FixedBytes(x.to_fixed_bytes().into()))
-                        .collect();
-                    Ok(ethers::abi::encode(&[Token::FixedArray(validator_tokens)]))
-                }
-                MetadataToken::Signatures => {
-                    let ordered_signatures = order_signatures(validators, &metadata.signatures);
-                    let threshold_signatures = &ordered_signatures[..threshold as usize];
-                    Ok(threshold_signatures.concat())
-                }
+                MetadataToken::Signatures => Ok(metadata
```

Comment:
If we're removing `order_signatures` and grabbing only `threshold` signatures, can you make sure we document somewhere where we get that guarantee? E.i. `rust/hyperlane-base/src/types/multisig.rs`'s `fetch_checkpoint` now must return exactly `threshold` signatures and in the same order as `validators`. It does this in practice but is never explicitly documented, hence why we had this logic here